tools/state-viewer/src/state_parts.rs

```diff
@@ -270,38 +279,50 @@ fn apply_state_parts(
         assert!(part_id < num_parts, "part_id: {}, num_parts: {}", part_id, num_parts);
```

Comment:
in the comment above: "Applying state as seen at the beginning of the specified epoch." -> "Loading state as seen at the beginning of the specified epoch."